packages/terra-badge/src/_mixins.scss

```diff
@@ -2,3 +2,34 @@
   background-color: map-get($color-map, background-color);
   color: map-get($color-map, color);
 }
+
+$default-theme: 'consumer' !default;
```

Comment:
Correct, we'd put this in [terra-mixins](https://github.com/cerner/terra-core/tree/master/packages/terra-mixins) and import it into each component `scss` file similar to how we did bidi mixins. 

I put it here just as a p.o.c. for how these mixins could work.